crates/host/src/metrics.rs

```diff
@@ -3,7 +3,9 @@ use std::time::Duration;
 
```

Comment:
When configuring timeout_ms, I found it confusing that I could not configure the value as a yaml number
```
              config:
                - name: default-http
                  properties:
                    address: 0.0.0.0:30000
                    timeout_ms: 300000
```

But instead had to configure it as a string
```
              config:
                - name: default-http
                  properties:
                    address: 0.0.0.0:30000
                    timeout_ms: '300000'
```

Reply:
Yes unfortunately this is a `wadm` issue, all configuration is assumed to be `string -> string`. 

I think we may want to fail validation of this if you use a non-string to avoid this confusion